packages/pds/src/app-view/services/feed/types.ts

```diff
@@ -1,9 +1,10 @@
-import { View as ViewImage } from '../../../lexicon/types/app/bsky/embed/images'
+import { View as ViewImages } from '../../../lexicon/types/app/bsky/embed/images'
 import { View as ViewExternal } from '../../../lexicon/types/app/bsky/embed/external'
 import { View as ViewRecord } from '../../../lexicon/types/app/bsky/embed/record'
+import { View as ViewComplexRecord } from '../../../lexicon/types/app/bsky/embed/recordWithMedia'
```

Comment:
not a big deal, but "complex record" hung around here

Reply:
Ah, thanks for nabbing that.  Should be patched-up.